### PR TITLE
Add definitions of latest releases

### PR DESCRIPTION
**Pull request by BanzaiMan**
Use Xdebug 3.0.0 for PHP 8.0.0

### CODE REVIEW

**Comment by BanzaiMan** [comment]
This includes all of the changes suggested by #651.

File: share/php-build/definitions/7.3.25 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+configure_option "--without-pear"
+configure_option "--with-gd"
+configure_option "--with-png-dir" "/usr"
+configure_option "--with-jpeg-dir" "/usr"
+configure_option "--enable-zip"
+
+install_package "https://secure.php.net/distributions/php-7.3.25.tar.bz2"
+install_xdebug "2.9.8"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/7.3.25:8]: incorrect xdebug version

File: share/php-build/definitions/7.4.13 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+configure_option "--enable-gd"
+configure_option "--with-jpeg"
+configure_option "--with-zip"
+
+install_package "https://secure.php.net/distributions/php-7.4.13.tar.bz2"
+install_xdebug "2.9.8"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/7.4.13:6]: incorrect xdebug version

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@ jobs:
           - '7.0.33'
           - '7.1.33'
           - '7.2.34'
-          - '7.3.24'
-          - '7.4.12'
+          - '7.3.25'
```
Comment (GrahamCampbell) [inline .github/workflows/ci.yml:25]: Please also replace the other occurrences of 7.3.25 and 7.4.13 in this file (yes, I know our current config needs improvement).

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
Actually, I don't think this is needed now. The original PR was improved, and merged, in the meanwhile.

**Pull request closed without merging**